config/init/common/lxc-net.in

```diff
@@ -20,10 +20,11 @@ LXC_DHCP_PING="true"
 LXC_DOMAIN=""
 LXC_USE_NFT="true"
 
-LXC_IPV6_ADDR=""
-LXC_IPV6_MASK=""
-LXC_IPV6_NETWORK=""
-LXC_IPV6_NAT="false"
+# IPv6 connectivity
+LXC_IPV6_ADDR="fc11:4514:1919:810::1"
+LXC_IPV6_MASK="96"
+LXC_IPV6_NETWORK="fc11:4514:1919:810::/96"
```

Comment:
I'd want this to be a /64 instead so that things like SLAAC can work properly.

Reply:
Change applied!